btservice/AndroidBluetoothServer.cpp

```diff
@@ -1,53 +1,214 @@
-/*
```

Comment:
im not sure about the "ethics" of moving the original author... i would think we would just leave it in here